src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/ServiceBusTriggerBinding.cs

```diff
@@ -30,30 +30,31 @@ internal class ServiceBusTriggerBinding : ITriggerBinding
         private readonly AccessRights _accessRights;
         private readonly ServiceBusConfiguration _config;
 
-        public ServiceBusTriggerBinding(string parameterName, Type parameterType, 
-            ITriggerDataArgumentBinding<BrokeredMessage> argumentBinding, ServiceBusAccount account, string queueName, AccessRights accessRights, ServiceBusConfiguration config)
+        public ServiceBusTriggerBinding(string parameterName, Type parameterType, ITriggerDataArgumentBinding<BrokeredMessage> argumentBinding, ServiceBusAccount account,
+            AccessRights accessRights, ServiceBusConfiguration config, string queueName)
+            : this(parameterName, parameterType, argumentBinding, account, accessRights, config)
         {
-            _parameterName = parameterName;
-            _converter = CreateConverter(parameterType);
```

Comment:
Here I'm removing duplicated constructor code to make this more maintainable. The actual bug was that we weren't setting _converter in the topic ctor, only the queue ctor. That caused the null ref exception that was reported.